### ISSUE TITLE
How to load a fine-tuned model and do predictions?

### ISSUE THREAD

**Issue by terenceau2** [huggingface/transformers#13969]
I have fine-tuned a Named entity recognition BERT model using the example here
https://github.com/huggingface/transformers/tree/master/examples/pytorch/token-classification

The command I used is:
 python3 run_ner.py --config_name (The bert model I chose) --tokenizer_name (The bert model I chose) --model_name_or_path (The bert model I chose) --train_file (My train file) --(my validation file)  --text_column_name Words --label_column_name Tags --output_dir output --do_train --do_eval 

I successfully obtained the output, which is  config.json and model.h5

With this, how can I load my model (the model.h5 above) and make a prediction, on an example sentence such as "I visited Github today"?

For the tutorial in the link above, the code has the "do_predict" option, but I do not know how to ask the "run_ner.py" code to load my fine-tuned model, and I want to see the predictions (instead of the code giving me a metrics of accuracy = xx%, precision = xx%.

Thanks.

**Comment by patil-suraj** [comment]
Hi, please use the [forum](https://discuss.huggingface.co/) to post such general questions, we use issues for bug reports and feature requests. Thank you!

**Comment by terenceau2** [comment]
> Hi, please use the [forum](https://discuss.huggingface.co/) to post such general questions, we use issues for bug reports and feature requests. Thank you!

Apologies, I am quite new here. Please close my post if necessary.
